### PR TITLE
feat: Implement AboutUs Section Fragment

### LINKED ISSUE #124: Missing About US Implementation.
**Actual Behaviour**
The About us section is blank currently in the activity.

**Expected Behaviour**
There should be the about us fragment implemented.

**Steps to reproduce it**

Navigate to the About us section from drop down menu.

**Would you like to work on the issue?**
yes

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #124 

**Changes**: The actual functional scrolling fragment implemented.

**Screenshot/s for the changes**:
Screenshot_20190329-220101

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]

### CODE REVIEW

**Comment by jddeep** [comment]
@mariobehling @iamareebjamal Please have a look. Thanks.

File: app/src/main/res/raw/aboutus.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The goal of the Neurolab project is to create an easy to use open hardware measurement headset device for brain waves that can be plugged into an Android smartphone and a software application and enables us to understand our brains better.
```
Comment (iamareebjamal) [inline app/src/main/res/raw/aboutus.txt]: Just include it in strings

**Comment by jddeep** [comment]
@iamareebjamal Please review now. Thanks!

File: app/src/main/res/layout/activity_about__us.xml [hunk 2]
```diff
@@ -7,5 +7,24 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+            <TextView
+                android:id="@+id/about_us_txt_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:padding="@dimen/about_txt_padding"
+                android:textColor="@android:color/black"
+                android:textSize="@dimen/about_text_size" />
```
Comment (iamareebjamal) [inline app/src/main/res/layout/activity_about__us.xml:27]: Directly load string here

File: app/src/main/java/io/neurolab/About_Us.java [hunk 3]
```diff
@@ -2,12 +2,21 @@
...
 public class About_Us extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_about__us);
+
+        TextView aboutUsTextView = findViewById(R.id.about_us_txt_view);
+
+        aboutUsTextView.setText(getResources().getString(R.string.about_us_content));
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/About_Us.java]: Remove

File: app/src/main/java/io/neurolab/About_Us.java [hunk 4]
```diff
@@ -2,12 +2,21 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.widget.TextView;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
 
 public class About_Us extends AppCompatActivity {
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/About_Us.java]: AboutUsActivity

**Comment by jddeep** [comment]
@iamareebjamal Please review now. Thanks.

**Comment by jddeep** [comment]
@iamareebjamal Please review and merge. Do suggest changes if any.

**Comment by iamareebjamal** [comment]
Peer review first

**Comment by jddeep** [comment]
@CoderVishalSehgal Do give me some reviews on this.

**Comment by vishal-sehgal** [comment]
@jddeep Fine from my side 👍

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**